in-market-test/v2/ansible/network-template.toml

```diff
@@ -0,0 +1,60 @@
+[[peers]]
+certificate = """
+-----BEGIN CERTIFICATE-----
+MIIBgDCCASagAwIBAgIIJIo0r8LAFEQwCgYIKoZIzj0EAwIwHTEbMBkGA1UEAwwS
```

Comment:
can we make those broken so we don't accidentally run these in prod?

Reply:
they should be broken because they are tied to local `draft.test` domains, but I can make that even more explicit but putting in a line at the top that says "NOT A REAL CERT" and mixing up some of the data further.